code/game/objects/items/devices/forcefieldprojector.dm

```diff
@@ -42,7 +46,16 @@
 	if(LAZYLEN(current_fields) >= max_fields)
 		to_chat(user, span_warning("[src] cannot sustain any more forcefields!"))
 		return
-
+	if(force_proj_busy)
+		to_chat(user, span_notice("[src] is busy creating a forcefield."))	
```

Comment:
```suggestion
		to_chat(user, span_notice("[src] is busy creating a forcefield."))
		return
```

Reply:
Make sure you don't have excess white space and you also need a return otherwise this won't do what you expect

Reply:
Thank you for catching that, making changes now.